cross-build/ethereum/utils.sh

```diff
@@ -373,3 +373,37 @@ function ct_ng_config_hack() {
 
 # ===========================================================================
 
+export -f contains
```

Comment:
export -f is to export functions, so they'll be accessible outside this script (I thought they were already but this was a silent bug!)